beetsplug/alternatives.py

```diff
@@ -347,35 +351,37 @@ def __init__(self, log, name, formats, lib, config):
         self.formats = [convert.ALIASES.get(f, f) for f in formats]
         self.convert_cmd, self.ext = convert.get_format(self.formats[0])
 
-    def converter(self):
+    @override
+    def _converter(self) -> "Worker":
         fs_lock = threading.Lock()
 
         def _convert(item):
             dest = self.destination(item)
             with fs_lock:
                 util.mkdirall(dest)
 
-            if self.should_transcode(item):
+            if self._should_transcode(item):
                 self._encode(self.convert_cmd, item.path, dest)
                 # Don't rely on the converter to write correct/complete tags.
                 item.write(path=dest)
             else:
                 self._log.debug("copying {0}".format(displayable_path(dest)))
                 util.copy(item.path, dest, replace=True)
             if self._embed:
-                self.sync_art(item, dest)
+                self._sync_art(item, dest)
             return item, dest
 
         return Worker(_convert, self.max_workers)
 
-    def destination(self, item: Item):
-        dest = super(ExternalConvert, self).destination(item)
```

Comment:
It's amazing how there are still remains of Python 2 times to be found; I would've been sure that we had removed them by now.